app/controllers/students_controller.rb

```diff
@@ -19,11 +29,21 @@ def create
 
     @student.save
     redirect_to students_path
-
   end
 
   def student_params
     params.require( :student).permit( :name)
   end
 
+ # DELETE /student/1
+  # DELETE /student/1.json
+  def destroy
+    @student.find(:id).destroy
+  #  respond_to do |format|
+  #    format.html { redirect_to student_url, notice: 'Student was successfully removed.' }
+  #    format.json { head :no_content }
+  #  end
+    redirect _to student_path
```

Comment:
Part of your problem is here. `redirect_to` shouldn't have a space inside of it. Also, `student_path` is the path to one particular student. Maybe you meant to use `students_path` (plural).

If you really did mean to use `student_path` you'll need to give it the ID or instance of a currently existing student. Normally, I'd say you could redirect to the `@student` you defined on line 41, but you just destroyed it, so you'd have to pick another one. I don't think `student_path` makes sense to use here in most cases.